src/ViewHelpers/LengthViewHelper.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file belongs to the package "TYPO3 Fluid".
+ * See LICENSE.txt that was shipped with this package.
+ */
+
+namespace TYPO3Fluid\Fluid\ViewHelpers;
+
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+
+/**
+ * The LengthViewHelper returns the length of a given string.
+ * Setting the character encoding is optional.
+ *
+ *
+ * Examples
+ * ========
+ *
+ * Simple example
+ * --------------
+ *
+ * ::
+ *
+ *      <f:length value="Hello, World!" />
+ *
+ * Output::
+ *
+ *      13
+ *
+ * Inline notation
+ * ---------------
+ *
+ * ::
+ *
+ *      {f:length(value: 'Hello, World!')}
```

Comment:
Could you add an example for the encoding argument?

Reply:
Should there also be a link to the possible encoding from php documentation?